webassembly/using_wasm_async.md

```diff
@@ -0,0 +1,40 @@
+## Using the Asynchronous WebAssembly APIs
```

Comment:
```suggestion
## Using the asynchronous WebAssembly APIs
```

Reply:
Really the WebAssembly instantiation APIs are all async. These APIs are just _streaming_. Maybe rename to `Using the streaming WebAssembly APIs`